src/Components/Notifications/NoticeBoard.tsx

```diff
@@ -55,13 +57,30 @@ export const NoticeBoard: any = () => {
     );
   } else {
     notices = (
-      <Card key="no-notice" className="my-4 rounded-lg">
-        <CardContent>
-          <div className="text-xl text-center semibold">
-            {t("no_notices_for_you")}
+      <div
+        className="max-w-sm rounded overflow-hidden shadow-lg "
+        style={{ marginLeft: "35%", marginTop: "45px" }}
+      >
+        <img className="w-full" src={bell} alt="Sunset in the mountains"></img>
+        <div className="px-6 py-4">
+          <div className="font-bold text-xl mb-2">
+            You Dont Have Any Notices
           </div>
-        </CardContent>
-      </Card>
+          <p className="text-gray-700 text-base">
+            Did you know: We could render a fun fact about health here
+          </p>
+        </div>
+        <div className="px-6 pt-4 pb-2">
+          <button
+            className="bg-green-500 text-white hover:bg-green-700 text-white font-bold py-2 px-4 rounded-full"
```

Comment:
Replace `green` with `primary`.
```suggestion
            className="bg-primary-500 text-white hover:bg-primary-700 text-white font-bold py-2 px-4 rounded-full"
```